docs/bucket/notifications/README.md

```diff
@@ -175,10 +176,14 @@ An example configuration for MQTT is shown below:
         "qos": 1,
         "clientId": "minio",
         "username": "",
-        "password": ""
+        "password": "",
+        "queueDir": ""
     }
 }
 ```
+MQTT supports persistent event store in the client-side. The persistent store will backup events incase if the broker goes down and replays it later when the mqtt broker comes back online. The event store can be configured by setting the directory path in `queueDir` field in the mqtt config. For eg, the `queueDir` can have `/tmp/events/mqtt`.
```

Comment:
How about, "The persistent store will backup events when the MQTT broker goes offline and replays it when the broker comes back online."?

And "the `queueDir` can be `/tmp/events`?

Reply:
Yes, changed it